docker-compose.yml

```diff
@@ -1,17 +1,21 @@
 version: '3'
+
 services:
+
   studio-app:
     build:
       context: .
-      dockerfile: Dockerfile.dev
+      dockerfile: docker/Dockerfile.dev
     image: learningequality/studio-app-dev
-    command: npm run devserver
+    command: make devserver
     depends_on:
       - minio
       - studio-postgres
       - studio-redis
     ports:
       - "8080:8080"
+    expose:
```

Comment:
The [wording for the `expose` direction](https://docs.docker.com/compose/compose-file/#expose) makes it sound like it's a subset of what's accomplished by the `ports`
> Expose ports without publishing them to the host machine - they’ll only be accessible to linked services.

Seems like it's _necessary_ to expose ports between different services?